kernel/include/stddef.h

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
I've added this file to handle the basic definition of `NULL` and `size_t` types, if we have an independent OS, it has to have all headers and declarations using a non-standard libc.